src/engine/DebugOverlays.EntityLabel.cs

```diff
@@ -47,7 +47,8 @@ private void UpdateEntityLabels()
                 {
                     if (microbe.Species != null)
                     {
-                        label.Text = $"[{microbe.Name}:{microbe.Species.Genus[0]}.{microbe.Species.Epithet.Left(4)}]";
+                        label.Text =
+                            $"[{microbe.Name}:{microbe.Species.Genus.Left(1)}.{microbe.Species.Epithet.Left(4)}]";
```

Comment:
Why does adding `Left` fixes the crash?

Reply:
Directly use [0] triggers an exception, but Left(1) returns string.empty if there isn't any char.

Reply:
When I coded this part, I assumed that the species may be less than four characters, so I used left(4), But I assume that there will be at least one character in genus, so I directly used [0].

Reply:
Ah okay.

Reply:
How does there exist species with no genus? Isn't that *very* invalid data?

Reply:
In multicellular editor, the cell on display.

Reply:
Ah okay, I suppose that is going to be different. I guess I'll accept this then here as this is a debugging feature.